components/api/gen/schema.bal

```diff
@@ -90,6 +90,17 @@ public type StringRecord record {
     string value;
 };
 
+public type OrgListResponse record {
+    int totalCount; 
```

Comment:
Shall we use count instead of totalCount ?. We have already used count for some of the implemented APIs